### ISSUE TITLE
There is no search functionality in the documentation

### ISSUE THREAD

**Issue by glyad** [elsa-workflows/elsa-website#45]
The fact that the documentation doesn't have a search box takes up too much expensive work time.

**Comment by sfmskywalker** [comment]
I agree, there should be a search feature. Docusaurus 2.0.0 (beta) seems to have a search feature so maybe it would be good to upgrade. Or maybe use a different solution (GitHub Wiki for example).
